source/js/Module/Event/Components/EventItem.js

```diff
@@ -1,29 +1,74 @@
 import PropTypes from 'prop-types';
+var today = new Date();
```

Comment:
- Move into getDateBadge function (no need to expose it in the outer scope)
- rename from today to date
- change to const